docs/release_notes/2024/july2024.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 1
+id: July_2024
+description: Release notes for July
+title: July 2024
+tags:
+  - release notes
+  - changelog
+  - july
+---
+
+## 3.3.8.\* (July 8,2024)
+
+- Prophecy Python libs version: 1.9.9
+- Prophecy Scala libs version: 8.0.22
+
+### Features {#Features337}
+
+#### Supporting maven dependencies for Livy
+
+Prophecy now supports adding Maven dependencies to Pipelines, which are automatically installed on a cluster during Pipeline attachment. This feature, previously available for Databricks, is now extended to Livy Fabrics.
+
+#### Airflow Copilot enhancements
+
+- **New Categories And Gems**: We have introduced new categories and Gems, including SFTP Sensor, SFTP to Snowflake, Tableau Extract, and SFTP to S3. The Gem Palette has been re-categorized with new sections for Data Transfer, Trigger/Notify, etc., to enhance Gem discovery.
+- **Simplified Model Gem**: The UX for the Model Gem in Airflow has been simplified. The Model Gem now automatically handles running the current working branch for the selected model and schedules the same upon release. Users can still choose to override these settings in the UI.
+
+More detailed documentation will be available soon.
+
+#### Pipeline Monitoring (beta)
+
+Pipeline Monitoring enables Spark Copilot users to monitor and debug their scheduled and interactive Pipeline runs through Prophecy. When enabled, Prophecy highlights the Gem causing the failure at runtime. Users can view log stack traces in the Prophecy UI to debug runtime issues faster. This feature is available for all interactive runs through the Prophecy UI and any scheduled Job runs.
```

Comment:
nit: Could you update the line to "When enabled, Prophecy highlights the Gem that causes the failure at runtime."